Directory.Build.props

```diff
@@ -0,0 +1,66 @@
+﻿<Project>
+    <PropertyGroup>
+        <Copyright>Copyright (c) Seth Reno. All rights reserved</Copyright>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/sethreno/ReGitLint</PackageProjectUrl>
+        <Authors>Seth Reno</Authors>
+        <Owners>Seth Reno</Owners>
+        <PackageTags>.net;c#;git;lint;ci</PackageTags>
+        <PackageReleaseNotes>https://github.com/sethreno/ReGitLint/releases</PackageReleaseNotes>
+        <PackageReadmeFile>readme.md</PackageReadmeFile>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <NeutralLanguage>en-US</NeutralLanguage>
+        <MinClientVersion>2.14</MinClientVersion>
+
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <RepositoryUrl>https://github.com/sethreno/ReGitLint</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <Product>$(AssemblyName) ($(TargetFramework))</Product>
+        <Version>6.1.0-alpha</Version>
+
+        <!--<Nullable>enable</Nullable>-->
+        <ImplicitUsings>enable</ImplicitUsings>
+        <LangVersion>latest</LangVersion>
+        <NoWarn>$(NoWarn);CS1591</NoWarn>
+        <Platform>AnyCPU</Platform>
+
+        <EnableNETAnalyzers>true</EnableNETAnalyzers>
+        <AnalysisLevel>latest</AnalysisLevel>
+        <AnalysisMode>Default</AnalysisMode>
+
+        <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+        <!--<SymbolPackageFormat>snupkg</SymbolPackageFormat>-->
+        <IncludeSymbols>true</IncludeSymbols>
+        <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+
+        <DesignTimeBuild>false</DesignTimeBuild>
+        <DebugType>portable</DebugType>
+        <DebugSymbols>true</DebugSymbols>
+
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <GeneratePackageOnBuild Condition=" '$(IsTestProject)' != 'true'">true</GeneratePackageOnBuild>
+        <PackageIcon>icon.png</PackageIcon>
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+        <Deterministic>true</Deterministic>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+        <DebugType>full</DebugType>
+        <DebugSymbols>true</DebugSymbols>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <None Include="$(MSBuildThisFileDirectory)\LICENSE" Pack="true" PackagePath="\" />
```

Comment:
Packing this increases the NuGet download size for no reason. Using `PackageLicenseExpression` is more efficient.

Reply:
PackageLicenseExpression alone is not enough. If you don't include the actual license in the nuget it doesn't cover it.

Reply:
According to https://learn.microsoft.com/en-us/nuget/reference/nuspec#license, it is. Adding the file is only needed when a non-standard license is used.